paper/paper.md

```diff
@@ -1,43 +1,204 @@
 ---
-title: "`cvCovEst`: Cross-validated covariance matrix estimation in `R`"
+title: "`cvCovEst`: Cross-validated covariance matrix estimator selection and evaluation in `R`"
 tags:
   - R
-  - covariance
+  - covariance matrix
   - cross-validation
+  - high-dimensional statistics
+  - loss-based estimation
+  - multivariate analysis
 authors:
   - name: Philippe Boileau
     orcid: 0000-0002-4850-2507
-    affiliation: 1
+    affiliation: "1, 2"
+  - name: Brian Collica
+    orcid: 0000-0003-1127-2557
+    affiliation: 3
   - name: Nima S. Hejazi
     orcid: 0000-0002-7127-2789
-    affiliation: 1, 2
+    affiliation: "1, 2"
   - name: Mark J. van der Laan
     orcid: 0000-0003-1432-5511
-    affiliation: 2, 3, 4
+    affiliation: "4, 3, 2"
   - name: Sandrine Dudoit
     orcid: 0000-0002-6069-8629
-    affiliation: 2, 3, 4
+    affiliation: "3, 4, 2"
 affiliations:
   - name: Graduate Group in Biostatistics, University of California, Berkeley
     index: 1
   - name: Center for Computational Biology, University of California, Berkeley
     index: 2
   - name: Department of Statistics, University of California, Berkeley
     index: 3
-  - name: Division of Epidemiology and Biostatistics, School of Public Health, University of California, Berkeley
+  - name: Division of Biostatistics, School of Public Health, University of California, Berkeley
     index: 4
-date: 01 September 2020
+date: 11 March 2021
 bibliography: paper.bib
 ---
 
 # Summary
 
-[TO FILL IN]
+Covariance matrices play fundamental roles in myriad statistical procedures.
+When the observations in a dataset far outnumber the features, asymptotic theory
+and empirical evidence have demonstrated the sample covariance matrix to be the
+optimal estimator of this parameter. This assertion does not hold when the
+number of observations is commensurate or smaller than the number of features.
+Consequently, statisticians have derived many novel covariance matrix estimators
+for the high-dimensional regime, often relying on additional assumptions about
+the parameter's structural characteristics (e.g., sparsity). While these
+estimators have greatly improved the ability to estimate covariance matrices in
+high-dimensional settings, objectively selecting the best estimator from among
+the many possible candidates remains a largely unaddressed challenge. The
+`cvCovEst` package addresses this methodological gap through its implementation
+of a cross-validated framework for covariance matrix estimator selection. This
+data-adaptive procedure's selections are asymptotically optimal under minimal
+assumptions, meaning they are equivalent to the selections one would make if the
+true data-generating processes were known [@vdl2003unified].
```

Comment:
```suggestion
data-adaptive procedure's selections are asymptotically optimal under minimal
assumptions -- in fact, they are equivalent to the selections that would be made if given access to full knowledge of the true data-generating processes (i.e., an oracle selector) [@vdl2003unified].
```
The initial wording felt awkward since the second clause following "meaning" was not directly implied by the preceding clause.